File: desktop/src/main/java/com/husseinamro/exchange/api/model/ExchangeRates.java
package com.husseinamro.exchange.api.model;

import com.google.gson.annotations.SerializedName;
public class ExchangeRates {
    @SerializedName("usd_to_lbp")
    public Float usdToLbp;
    @SerializedName("lbp_to_usd")
    public Float lbpToUsd;
}

File: android/app/src/main/java/com/MounirKhalil/currencyexchange/api/model/ExchangeRates.kt
package com.MounirKhalil.currencyexchange.api.model

import com.google.gson.annotations.SerializedName
class ExchangeRates {
    @SerializedName("usd_to_lbp")
    var usdToLbp: Float? = null
    @SerializedName("lbp_to_usd")
    var lbpToUsd: Float? = null
}
